source/client/options_impl.cc

```diff
@@ -223,8 +237,8 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv) {
       "uri format", cmd);
   TCLAP::SwitchArg h2_use_multiple_connections(
       "", "experimental-h2-use-multiple-connections",
-      "Use experimental HTTP/2 pool which will use multiple connections. WARNING: feature may be "
-      "removed or changed in the future!",
+      "DO NOT USE: This options is deprecated, if this behavior is desired, set "
```

Comment:
nit: s/options/option/

Reply:
Done in c30393630fcaa1f88337537fe6aa83adaaaedb28.